test/jsonapi/view_test.exs

```diff
@@ -33,6 +35,14 @@ defmodule JSONAPI.ViewTest do
     assert PostView.url_for_rel(%{id: 1}, "comments", %Plug.Conn{}) == "http://www.example.com/api/posts/1/relationships/comments"
   end
 
+  test "url_for/2 with custom host" do
+    assert CommentView.url_for(nil, nil) == "www.notexample.com/api/comments"
```

Comment:
The `CommentView` module is only included for Phoenix tests, will this bomb out in the event `tags[:compile_phoenix]` is not set?